src/mame/video/decbac06.cpp

```diff
@@ -488,7 +488,7 @@ WRITE8_MEMBER( deco_bac06_device::pf_control1_8bit_w )
 		if (offset&1)
 			pf_control_1_w(space,offset/2,data,0x00ff);
 		else
-			pf_control_1_w(space,offset/2,data,0x00ff);
```

Comment:
This can't possibly be right - data won't overlap mask.  Also, are you sure about which lane is which?  (Same for other change in this file.)

Reply:
Looking at it I am not sure what is right anymore - feels a bit all over the place. I will just revert it.